### DISCUSSION TITLE
how can i use "alias" in SCSS?

### DISCUSSION THREAD

**Discussion by qianjiachun** [remix-run/remix#2552] [Q&A] [answered]
`import "~/**"` does not work. So I have to use a lot of `..`  

how can i use "alias" in SCSS?🙏

**Comment by sergiodxa** [comment, marked as answer]
> `import "~/**"` does not work. So I have to use a lot of `..`
> 
> how can i use "alias" in SCSS?🙏

Remix does nothing with your SCSS code so you will have to see how to setup that on your SCSS compiler yourself.

I recommend you to ask on the repo of Sass/SCSS
